### DISCUSSION TITLE
Branch Not Found dependabot/npm_and_yarn/tinacms-1.6.7

### DISCUSSION THREAD

**Discussion by GA8R137** [tinacms/tinacms#4683] [Help]
Hi, 
I am trying to use Tina cloud, with Vercel and Github
After hours of failed attempts the site is working but when I am trying to edit the demo site I get this error.
"Branch Not Found
The current branch (dependabot/npm_and_yarn/tinacms-1.6.7) has either been merged or deleted."

I found a setting where I can setup other versions of dependabot, 
2.0.0 
1,5.x
main

I tried the 3, also I tried in "advanced settings" to "export branch" or "refresh webhooks" and many other things but this error block me to enter in admin mode.  

what can I do?

**Comment by kldavis4** [comment]
Why are you editing on the dependabot branch?
